api/middleware/index.js

```diff
@@ -0,0 +1,46 @@
+module.exports = function(req, res, next) {
+    var jwt = require('jsonwebtoken')
+
+    // Bearer token and username
+    const bearer_header = req.headers.authorization // Bearer <token>
+    const from = req.headers.from
+
+    if (bearer_header == null || from == null) {
+        return res.status(400).set({
+            "Content-Type": "application/json",
+        }).send({ 
+            message: "Header is required" 
+        })
+    }
+
+    // Split token from bearer header
+    const bearer = bearer_header.split(' ')
+    const bearer_token = bearer[1] // <token>
+
+    // Verify token
+    jwt.verify(bearer_token, 'secret', function(err, decoded) {
```

Comment:
Can we camelCase the variable names please? Just going along with the Javascript style